pandas/io/gbq.py

```diff
@@ -682,6 +686,13 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
 
         .. versionadded:: 0.19.0
 
+    **kwargs: Arbitrary keyword arguments
```

Comment:
@jreback sorry for big response but it doesn't still clear for me:
1) The things I worry about are arguments ```query``` and ```dialect```. It is become redunt.
```
config = {
  'query' : {
    'query':  'select 1',
    'useLegacySql': dialect == 'legacy'
  }
}
read_gbq(query, dialect, configuration=config)
```
 So, I am going to use such logic: if query not specified in config I will use query parameter of read_gbq. But if query or dialect specified in config do I must throw an exception?

2) As I got your idea configuration parameter should be like

```
config = {
  'query' : {
    "useQueryCache": False,
  }
}
```
Not like:
```
config = {
  '"useQueryCache": False
}
```
And in this case I could pass:
```
config = {'load' : ....}
```
what I should do with query parameter of function ```read_gbq```? just skip it inside the logic.